pkg/kubecost/allocation.go

```diff
@@ -1018,18 +1015,14 @@ func (as *AllocationSet) AggregateBy(aggregateBy []string, options *AllocationAg
 		for _, alloc := range shareSet.allocations {
 			idleKey, err := alloc.getIdleKey(options)
 			if err != nil {
-				log.DedupedWarningf(5, "AllocationSet.AggregateBy: missing idleKey for allocation: %s", alloc.Name)
-				continue
+				log.DedupedWarningf(3, "AllocationSet.AggregateBy: missing idleKey for allocation: %s", alloc.Name)
```

Comment:
So, if we aren't continuing any longer, then what happens? How does this affect idle by cluster?

Reply:
So all of these continues actually came from the last PR on this [802](https://github.com/kubecost/cost-model/pull/802/files) and before they actually returned. The behavior from cluster id that you are talking about preserving is a blank allocations screen with an error in the network tab. I don't think that was the right behavior to begin with but it also stood in the way of this feature so I changed it.

Reply:
Okay, thanks for the context.